# Make sure this patch (of 'configure') is run after the patch of
# 'configure.in' to avoid triggering any 'configure: configure.in' make
# rules for people with autoconf.
#
--- configure-dist	Wed Jun  4 22:21:48 1997
+++ configure	Sat Jun 21 10:16:23 1997
@@ -3738,8 +3738,8 @@
 
 
 eval "nmhbin=${bindir}"; eval "nmhbin2=${nmhbin}"
-eval "nmhetc=${etcdir}"; eval "nmhetc2=${nmhetc}"
-eval "nmhlib=${libdir}"; eval "nmhlib2=${nmhlib}"
+eval "nmhetc=${sysconfdir}"; eval "nmhetc2=${nmhetc}"
+eval "nmhlib=${libexecdir}"; eval "nmhlib2=${nmhlib}"
 eval "nmhman=${mandir}"
 
 echo "
